apps/src/applab/designElements/ImagePickerPropertyRow.jsx

```diff
@@ -1,4 +1,5 @@
-var showAssetManager = require('../../assetManagement/show.js');
+/* global dashboard */
```

Comment:
I don't like adding this global, but the `dashboard` namespace is the main way we expose features from `code-studio` to `apps`.  (This is also why mocha tests are failing.)